afs/views/analysis_area_and_sample_taking.py

```diff
@@ -173,6 +174,11 @@ def post(self, request):
         part_identifier_assignment_tile_data = JSONDeserializer().deserialize(request.POST.get("partIdentifierAssignmentTileData"))
         part_identifier_assignment_tile_id = request.POST.get("partIdentifierAssignmentTileId") or None
         name = request.POST.get("analysisAreaName")
+        print("++++++++++++++++++++++")
+        print(name)
+        print("++++++++++++++++++++++")
```

Comment:
Maybe these print statements can be removed

Reply:
thanks